application/src/main/java/com/sanction/lightning/LightningConfiguration.java

```diff
@@ -29,4 +30,13 @@ public ThunderConfiguration getThunderConfiguration() {
   public final List<Key> getApprovedKeys() {
     return approvedKeys;
   }
+
+  @NotNull
+  @Valid
+  @JsonProperty("facebook-auth")
+  private final FacebookAuth facebookAuth = null;
+
+  public final FacebookAuth getFacebookAuth() {
```

Comment:
This method doesn't need to be final. Actually, I don't know why the 'getApprovedKeys()' method is final either.